packages/connectors/connector-patreon/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# @logto/connector-patreon
+
+## 0.0.1
+
+### Major Changes
+
+- Initial release of the Patreon connector.
+
+  This release introduces the Logto connector for Patreon, enabling social sign-in using Patreon accounts. It supports OAuth 2.0 authentication flow, fetching user information, and handling errors gracefully.
+
+### Features
+
+- **OAuth 2.0 Authentication**: Support for OAuth 2.0 authentication flow with Patreon.
+- **User Information Retrieval**: Fetches user details such as full name, email, profile URL, and avatar.
+- **Error Handling**: Graceful handling of OAuth errors, including token exchange failures and user-denied permissions.
+- **Configurable Scope**: Allows customization of OAuth scopes to access different levels of user information.
```

Comment:
this can be move to changeset and let the cli generate the file

Reply:
updated.